composer.json

```diff
@@ -43,6 +42,7 @@
         "doctrine/coding-standard": "^8.2.0",
         "infection/infection": "^0.19.2",
         "malukenho/mcbumpface": "^1.1.5",
+        "ocramius/package-versions": "^1.9",
```

Comment:
`ocramius/package-versions` is not a dev dependency. Other than that it looks fine.

Reply:
Fixed. Thanks! 👍🏻